omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -264,10 +268,17 @@ public String post(@RequestParam(value = "username", required = false) String us
 					}
 					
 					//If there is a single login location, default to that
+					Boolean clientTimezonePropriety = false;
+					if(Context.isAuthenticated()){
+						clientTimezonePropriety = StringUtils.isBlank(Context.getAuthenticatedUser().getUserProperty("clientTimezone"));
```

Comment:
Did you intend to type clientTimezoneProperty?

Reply:
Yes, its a typo.